NEWS.md

```diff
@@ -5,6 +5,17 @@ shinydashboard 0.7.0
 
 ### New features
 
+* Shiny async support: render functions can now handle promises. Instead of returning a `valueBox` to `renderValueBox`, return a promise that resolves to a `valueBox` (for example).
+
+### Bug fixes
+
+shinydashboard 0.7.0
```

Comment:
Can you bump the previous `shinydashboard 0.7.0` to shinydashboard 0.7.0.9000`? (I'm not able to comment on that line because of how the diff was inferred.)